File: Atv4_Repeticao/Senha/senha.c
#include <stdio.h>
#include <string.h>

int main() {
    char senha[20];
    char senhaCorreta[] = "1234";

    do {
        printf("Digite a senha: ");
        scanf("%s", senha);
        if (strcmp(senha, senhaCorreta) != 0)
            printf("Senha incorreta!\n");
    } while (strcmp(senha, senhaCorreta) != 0);

    printf("Senha correta!\n");
    return 0;
}

File: Atv4_Repeticao/Senha/senha.cs
using System;

class Program {
    static void Main() {
        string senhaCorreta = "1234";
        string senha;

        do {
            Console.Write("Digite a senha: ");
            senha = Console.ReadLine();

            if (senha != senhaCorreta)
                Console.WriteLine("Senha incorreta!");
        } while (senha != senhaCorreta);

        Console.WriteLine("Senha correta!");
    }
}